### PR TITLE
Parameter Scan

### PR DESCRIPTION
**Pull request by BryanRumsey**
## New Features
- New parameter scan workflow allows users to sweep through more than 2 parameters and view time series plots.
- Added validation that prevents users from starting parameter sweep/scan jobs if there are no parameters.

## Updates
- Updated the Example SIR Epidemic project

## Bug Fixes
- Fixed bug with event assignments
- Fixed default plot request key for time series visualizations.
- Fixed bug with ODE jobs and previews.
- Fixed bug with visualization plots when an error was previously displayed.
- Fixed bug with job info not being updated when the job is completed.

### CODE REVIEW

File: stochss/handlers/util/parameter_scan.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            for val in param['range']:
+                variables[param['parameter']] = val
+                self.__run(index=index, variables=variables, verbose=verbose)
+        else:
+            tmp_mdl = self.__setup_model(variables=variables)
+            result_key = self.__get_result_key(variables=variables)
+            if verbose:
+                message = f'running {result_key.replace(":", "=").replace(",", ", ")}'
+                print(message)
+            tmp_result = tmp_mdl.run(**self.settings)
```
Comment (briandrawert) [inline stochss/handlers/util/parameter_scan.py:70]: Add try/catch here

Comment (briandrawert) [reply stochss/handlers/util/parameter_scan.py:70]: Put the contents of the caught exception in the user error logs

**Review by briandrawert — APPROVED** [review]

**Review by seanebum — APPROVED** [review]

File: client/templates/includes/sweepParameterRange.pug [hunk 2]
```diff
@@ -7,12 +7,12 @@ div
 
     div.inline.mr-2=this.model.name + ":"
 
-    div.inline(data-hook=this.model.elementID + "-current-value")=this.parameter.expression
+    div.inline(data-hook=this.model.elementID + "-current-value")=this.model.range[0]
```
Comment (jtcooper10) [inline client/templates/includes/sweepParameterRange.pug:10]: When the current value is a long decimal like `0.07000000000001`, it spills off the screen at certain screen sizes. May want to round the display?

**Review by jtcooper10 — APPROVED** [review]
Works great for me, tested on Fedora Linux and Windows WSL2, both Chrome and Edge.

**Review by ethangreen-dev — APPROVED** [review]
Same here -- runs and works well on my machine.

**Pull request closed without merging**